test/check-application

```diff
@@ -536,17 +538,26 @@ class TestNavigator(testlib.MachineCase):
         # Create file symbolic link
         self.create_link(b, "test", "test-sym", "symbolic", 1)
         b.wait_visible("[data-item='test-sym']")
-        self.assertEqual(m.execute("ls -l /home/admin/test-sym")[0:23], "lrwxrwxrwx. 1 root root")
+        if any(image in m.image for image in DISTROS_WITH_SELINUX):
+            selinux_dot = "."
+        else:
+            selinux_dot = ""
+
+        def ls_output(filename):
+            output = m.execute(f"ls -l {filename}")[0:23]
+            return output.strip()
+
+        self.assertEqual(ls_output("/home/admin/test-sym"), f"lrwxrwxrwx{selinux_dot} 1 root root")
```

Comment:
That `DISTROS_WITH_SELINUX` is rather clunky -- why not `assertRegexp` with `"lrwxrwxrwx.* 1 root root"`?

Reply:
Agreed, nice suggestion!